service/ui.go

```diff
@@ -240,9 +240,8 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 	app.AddRoute("/perfdiscovery/").Wrap(needsLogin, needsContext).Handler(uis.perfdiscoveryPage).Get()
 	app.AddRoute("/perfdiscovery/{project_id}").Wrap(needsLogin, needsContext).Handler(uis.perfdiscoveryPage).Get()
 
-	// Signal Processing page
-	app.AddRoute("/signal-processing/").Wrap(needsLogin, needsContext).Handler(uis.signalProcessingPage).Get()
-	app.AddRoute("/signal-processing/{project_id}").Wrap(needsLogin, needsContext).Handler(uis.signalProcessingPage).Get()
+	// Signal Processing page (UI-routing enabled for this route)
+	app.AddRoute("/perf-bb/{_}/{project_id}").Wrap(needsLogin, needsContext).Handler(uis.signalProcessingPage).Get()
```

Comment:
Neither of these variables are used in the route handler - are they needed? Also the first one should have a name